bigbluebutton-tests/playwright/options/options.js

```diff
@@ -67,7 +67,7 @@ class Options extends MultiUsers {
     await this.modPage.waitAndClick(e.modalConfirmButton);
 
     const modPageLocator = this.modPage.getLocator('body');
-    await this.modPage.page.setViewportSize({ width: 1366, height: 768 });
+    await this.modPage.setViewPortSize(1366, 768);
```

Comment:
As this viewport size is being used multiple times, we could have a (helper, maybe?) function with this value and then just call it